### PR TITLE
Added Error when no content

### PR DESCRIPTION
**Pull request by pfreiberger**
The code was raising a generic exception whenever status code is not 200. However, often the code is 204 which indicates that the request wa ssuccessful but no contet is available. I added a few lines to raise a NoContentException is one can easily catch these types of exceptions

### CODE REVIEW

File: elexon/elexon.py [hunk 1]
```diff
@@ -180,6 +179,8 @@ def _check_error(self, metadata) -> None:
         r_httpCode = metadata.find('httpCode').text
         r_errorType = metadata.find('errorType').text
         r_description = metadata.find('description').text
+        if r_httpCode == "204":
```
Comment (michaelkavanagh) [inline elexon/elexon.py]: Please can you change these to single quotes for consistency?

**Review by michaelkavanagh — CHANGES_REQUESTED** [review]
Hi!

Thank you for your PR, in general these changes look good to me. 

I've left one small formatting request, but otherwise I am happy to merge this.

Thanks.

File: elexon/elexon.py (hunk 1 above)
Comment (pfreiberger) [reply elexon/elexon.py]: Just pushed the change below

**Comment by michaelkavanagh** [comment]
Great, thank you for your contribution!

**Pull request merged**